rites/roman1969/__tests__/calendar-builder.test.ts

```diff
@@ -1,13 +1,11 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
-import { England_En } from 'romcal/dist/bundles/england';
-import { GeneralRoman_En } from 'romcal/dist/bundles/general-roman';
-import { Germany_En } from 'romcal/dist/bundles/germany';
-import { Hungary_En } from 'romcal/dist/bundles/hungary';
-import { Ireland_En } from 'romcal/dist/bundles/ireland';
-import { Slovakia_Sk } from 'romcal/dist/bundles/slovakia';
-
-import { CalendarScope, Id, LiturgicalCalendar, LiturgicalDay, LiturgicalDayDef, Period, Romcal, Season } from '../lib';
-import { dateDifference } from '../lib/utils/dates';
+import { England_En } from '../dist/bundles/england';
+import { GeneralRoman_En } from '../dist/bundles/general-roman';
+import { Germany_En } from '../dist/bundles/germany';
+import { Hungary_En } from '../dist/bundles/hungary';
+import { Ireland_En } from '../dist/bundles/ireland';
+import { Slovakia_Sk } from '../dist/bundles/slovakia';
+import { CalendarScope, Id, LiturgicalCalendar, LiturgicalDay, LiturgicalDayDef, Period, Romcal, Season } from '../src';
```

Comment:
as above, we should alias `src` and `src/utils` unless we're just going to reference `@src` all the time, that way if we move files, we don't have to manually update the relative pathing

Reply:
I introduced paths alias like `@dist/rite-roman1969/bundles/germany` and `@src/rite-roman1969`. We'll see how it works in practice, or if there are better suggestions.